Core/Requirement/RequirementFactory.cs

```diff
@@ -90,7 +90,10 @@ public Requirement GetRequirement(string name, string requirement)
             if (BuffDictionary.Count == 0)
             {
                 BuffDictionary = new Dictionary<string, Func<bool>>
-                {
+                {   
+                    // Target
+                    {  "hastarget", ()=> playerReader.HasTarget },
+                    {  "targetalive", ()=> !playerReader.PlayerBitValues.TargetIsDead },
```

Comment:
Why do you expose these conditions if no class profile are using them? What point does they serve.

On top of the you have to update the readme.md in root, in order to other know about such changes